src/resolvers/Mutation/createVenue.ts

```diff
@@ -111,7 +111,7 @@ export const createVenue: MutationResolvers["createVenue"] = async (
   }
 
   // Check if the venue name provided is an empty string.
-  if (!args.data?.name ?? "") {
+  if (!args.data?.name) {
```

Comment:
```suggestion
if (!(args.data?.name ?? "")) {
```

try this, 

Here we wrapped `args.data?.name ?? ""` in parentheses before applying the logical `NOT ! `operator.